src/app/screens/Dashboard/screens/UseState/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import ScreenView from '~components/ScreenView';
+
+import SquareFunctional from './components/SquareFunctional';
+import SquareClassComponent from './components/SquareClassComponent';
+
+function UseState() {
+  return (
+    <ScreenView title="useState example">
+      <SquareFunctional />
+      <SquareClassComponent />
```

Comment:
I'm not sure about showing class components 🤔 , we could leave them for now if you want.

Reply:
We can keep it but not speak about it